selfdrive/car/nissan/interface.py

```diff
@@ -35,6 +37,8 @@ def _update(self, c):
     be.type = car.CarState.ButtonEvent.Type.accelCruise
     buttonEvents.append(be)
```

Comment:
what is this?!

Reply:
This is already removed in upstream. Will sync now.

Reply:
I see! https://github.com/commaai/openpilot/pull/31768